src/kata7.getEmployerRole.js

```diff
@@ -1,3 +1,8 @@
-const getEmployerRole = (employeeName, employees) => {};
+const getEmployerRole = (employeeName, employees) => {
+  const anEmployee = employees.find(
+    (employee) => employee.name === employeeName
+  );
+  return anEmployee.role;
+};
```

Comment:
As above this can be simplified into a readable one liner:
```
const getEmployerRole = (employeeName, employees) =>
  employees.find((employee) => employee.name === employeeName).role;
```